index.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+
+const {
+  getSurveyFromRegistry,
+  getSurveyFromPublisher,
+  insertIntoSurveyRegister
```

Comment:
Should this be called `insertIntoSurveyRegistry` to be consistent with `getSurveyFromRegistry`?